development/image-guard/README.md

```diff
@@ -0,0 +1,28 @@
+# Image-guard
+This directory contains source code for the Image-guard admission controller.
+
+After installing the chart Image-guard performs the following tasks:
+* Monitor and log used docker images in the pods
+* Optionally check and allow usage only for whitelisted registries
+
+## Requirements
+To deploy the application you need the following:
+* helm 3
+* K8s cluster version at least 1.16
+
+## Installation
+To install image-guard use the following command using your k8s cluster context:
```

Comment:
```suggestion
To install image-guard, use the following command using your Kubernetes cluster context:
```

Reply:
`Image-guard` or `image-guard` or maybe `Image Guard`? 🤔

Reply:
I think it's best to stay with using `image-guard` as it's been loosely named to use it as name for the deployment.